createMeanyButelkaObjects.cls

```diff
@@ -0,0 +1,44 @@
+public  class createMeanyButelkaObjects {
+Integer howManyBottleToCreate ;
+public static Integer bottleNumber=0;
+
+public static void createMoreBottleObjects (Integer howManyBottleToCreate){
+
+        List<butelka> bottleObjectsList = new List <butelka>();
+        
+            for(Integer i=0; i< howManyBottleToCreate; i++){
+                Integer newBottleNumber = bottleNumber++;
+                butelka myBootle = new butelka(newBottleNumber, i+5,i+3*3);
+                bottleObjectsList.add(myBootle);
+        
+        }
+        System.debug( bottleObjectsList);
+
+        
+
+    for(butelka bott:bottleObjectsList){
+        
```

Comment:
Do takiego indexowania i sprawdzania kolejnosci bardziej nadał by siępętla iterujaca po I typu for (Integer i=0 itd itp i odwoływanie się do butelek, ale to kombinowanie i musialbym zrozumiec o co Ci chodzilo tutaj, ale generalnie jest git